index.html

```diff
@@ -20,7 +20,12 @@
       <footer>
         <!-- {% include banner.html %}  -->
         <div class="bounds">
-          <section class="hero-footer__section">
+          <section class="hero-footer__section" itemscope itemtype="http://schema.org/SoftwareApplication" >
+                <meta itemprop="ApplicationCategory"  content="hybrid mobile applications, web development"/>
```

Comment:
These indents still need addressing; they should be just inside the parent `section` and at the same level with the following `h2` tag.